### DISCUSSION TITLE
Hi, guys I have a problem for using react-router-dom

### DISCUSSION THREAD

**Discussion by Fandyfr** [remix-run/react-router#11729] [Q&A]
![image](https://github.com/remix-run/react-router/assets/135007919/f8b06a64-2869-4326-953a-28959b924240)

```
ERROR in ./node_modules/react-router-dom/dist/index.js 14:0-803
export 'AbortedDeferredError' (reexported as 'AbortedDeferredError') was not found in 'react-router' (possible exports: Await, MemoryRouter, Navigate, Outlet, Route, Router, RouterProvider, Routes, UNSAFE_DataRouterContext, UNSAFE_DataRouterStateContext,
UNSAFE_LocationContext, UNSAFE_NavigationContext, UNSAFE_RouteContext, UNSAFE_mapRouteProperties, UNSAFE_useRouteId,
UNSAFE_useRoutesImpl, createMemoryRouter, createRoutesFromChildren, createRoutesFromElements, renderMatches, useActionData,
useAsyncError, useAsyncValue, useBlocker, useHref, useInRouterContext, useLoaderData, useLocation, useMatch, useMatches,
useNavigate, useNavigation, useNavigationType, useOutlet, useOutletContext, useParams, useResolvedPath, useRevalidator,
useRouteError, useRouteLoaderData, useRoutes)
```
even though I checked, removed and re-installed and npm clean --force and installed again, it's still like that 🤷‍♂️

**Comment by knoguchi** [comment]
I deleted node_modules and npm install again.  I know it sounds like a random attempt.  Anyway it fixed my issue.  I guess it's a some kind of version incompatibility.  I didn't bother to look into details.

**Reply by Fandyfr** [reply]
still the same, I have removed the node modules, it's the same

**Comment by zdunecki** [comment]
In my case it was caused by having remix in my dependencies. 

I also had to delete `react-router-dom` ([manual / template installation ](https://reactrouter.com/dev/guides/start/installation)does not  have it too).
